app/dependency_file_updaters/node.rb

```diff
@@ -59,13 +57,14 @@ def updated_package_json_content
         JSON.pretty_generate(parsed_content) + "\n"
     end
 
-    def updated_shrinkwrap_content
-      return @updated_shrinkwrap_content if @updated_shrinkwrap_content
+    def updated_yarn_lock_content
+      return @updated_yarn_lock_content if @updated_yarn_lock_content
       SharedHelpers.in_a_temporary_directory do |dir|
+        File.write(File.join(dir, "yarn.lock"), @yarn_lock.content)
         File.write(File.join(dir, "package.json"), updated_package_json_content)
-        `cd #{dir} && npm i --silent && npm shrinkwrap --silent`
-        @updated_shrinkwrap_content =
-          File.read(File.join(dir, "npm-shrinkwrap.json"))
+        puts `cd #{dir} && yarn install --ignore-scripts`
```

Comment:
Returns the output of the shell command. I've added a `puts` here so we see it in logs, but not too fussed otherwise.